redis/Config.uk

```diff
@@ -0,0 +1,32 @@
+# Configure Redis server application for a build that uses initrd to pass the root
+# filesystem and a bridge networking.
```

Comment:
Rephrase to: " Configure Redis server application for a build with bridged networking and using initrd to pass the root filesystem."